### PR TITLE
Added real data to punchcard

### PR DESCRIPTION
**Pull request by borek2**
Added real data to the punch card WORKING 💃

### CODE REVIEW

File: src/modules/punch-card.js [hunk 1]
```diff
@@ -55,6 +55,7 @@ define(function () {
             "required": true
         }],
         body: function (res) {
+            console.log(res[0]);
```
Comment (lvdoorn) [inline src/modules/punch-card.js]: Maybe we should not put these in the master.

File: src/modules/punch-card.js [hunk 2]
```diff
@@ -86,12 +90,8 @@ define(function () {
...
 
             var g = d3.select(document.createElementNS(d3.ns.prefix.svg, "g"));
-            var dummyData = [
-                { start: "2016-06-06T21:00:00.529Z", end: "2016-06-07T22:30:00.529Z" },
-                { start: "2016-06-06T05:00:00.529Z", end: "2016-06-06T09:00:00.529Z" },
-                { start: "2016-06-07T11:00:00.529Z", end: "2016-06-07T14:00:00.529Z" },
-                { start: "2016-06-08T12:00:00.529Z", end: "2016-06-08T15:00:00.529Z" },
-            ];
+            var dummyData = res[0];
+            console.log(dummyData);
```
Comment (mboom) [inline src/modules/punch-card.js]: Maybe rename dummyData and remove console log.

File: src/modules/punch-card.js (hunk 1 above)
Comment (agudek) [inline src/modules/punch-card.js]: Agree

File: src/modules/punch-card.js [hunk 3]
```diff
@@ -86,12 +90,8 @@ define(function () {
...
             var transformedData = dummyData.map(function (item) {
```
Comment (mboom) [inline src/modules/punch-card.js]: or shorter: use res[0].map(...

**Pull request merged**